client/src/app/services/manage-members.service.ts

```diff
@@ -20,4 +20,9 @@ export class ManageMembersService {
   {
     return this.http.post(`${this.BASEURL}/getSomeClients/`, JSON.stringify(name));
   }
+
+  modifyClientInfo(category, field, id): Observable<any>
+  {
+    return this.http.post(`${this.BASEURL}/modifyClients/`, JSON.stringify({'category': category, 'field': field, 'id': id}));
```

Comment:
Instead of doing "JSON.stringify({'category': category, 'field': field, 'id': id})" you can just create an object called 'body' and add all those properties to it and then pass the body instead of that statement.